bears/uploadTool.py

```diff
@@ -0,0 +1,74 @@
+import argparse
+import glob
+import os
+import shutil
+from string import Template
+import subprocess
+import sys
+
+from coalib.collecting.Importers import iimport_objects
+
+
+def touch(file_name):
+    """
+    Creates an empty file. An existing file remains untouched.
+
+    :param file_name: Name of the file.
+    """
+    if not os.path.exists(file_name):
+        open(file_name, 'w').close()
+
+
+def main():
+    parser = argparse.ArgumentParser(
+        description='This tool uploads all the bears to PyPi in separate'
+            'packages.')
+    parser.add_argument('-u', '--upload', help='Upload the bears on PyPi',
+                        action='store_true')
+    args = parser.parse_args()
+
+    os.makedirs('upload', exist_ok=True)
+
+    bears_full_filenames = glob.glob('**/*Bear.py')
+
+    for bear_file_name in bears_full_filenames:
+        file_name = bear_file_name
+        bear_name = os.path.splitext(os.path.basename(bear_file_name))[0]
+        bear_object = next(iimport_objects(file_name, attributes='kind',
+                                           local=True))
+        upload_bear_folder = os.path.join('upload', bear_name, bear_name)
+        os.makedirs(upload_bear_folder, exist_ok=True)
+        touch(os.path.join(upload_bear_folder, '__init__.py'))
+        shutil.copyfile(file_name, os.path.join(upload_bear_folder,
+                                                bear_name + '.py'))
+
+        substitution_dict = {'NAME': bear_name,
+                             'VERSION': '1.03',  # testing purposes
```

Comment:
you mark it as for testing explicitly, still you want to merge this code into production code. Just import version from bears and take that one?